zanata-common-api/src/main/java/org/zanata/rest/client/IGlossaryResource.java

```diff
@@ -55,6 +98,13 @@
 
     @Override
     @DELETE
+    @Path("/{locale}/{resId}")
+    public ClientResponse<String> deleteGlossary(
+        @PathParam("locale") LocaleId locale, @PathParam("resId") String resId);
```

Comment:
let's delete the first (existing) deleteGlossary because of its misguided `@Path("/{locale}")`

this one should be named deleteEntry and it should use `@Path("/entries/{resId}")`